guidelines/index.html

```diff
@@ -533,6 +535,7 @@ <h2>Statement of Partial Conformance - Language</h2>
 
             <section class="informative" id="privacy-summary">
                 <h2>Privacy Considerations</h2>
+                <p>Success Criteria within this specification which the Working Group has identified possible implications for privacy, either by providing protections for end users or which are important for web site providers to take in to consideration when implementing features designed to protect user privacy, are listed below. This list reflects the current understanding of the Working Group but other Success Criteria may have privacy implications that the Working Group is not aware of at the time of publishing.</p>
```

Comment:
Maybe nitpicky, but the "are listed below" (here and in the other paragraph for security) look very lost/far away from the first part of the sentence, because of that massive sub-clause in the middle. Might be easier to read if this is turned around, a la "Listed below are Success Criteria..."

Reply:
Sure. I'll leave that to the editors.